@hubblecommerce/hubble/core/components/swComponents/slots/buy-box-slot.vue

```diff
@@ -1,6 +1,6 @@
 <template>
     <div :class="elementClass">
-        <product-detail-buybox v-if="product !== null" :data-product="product" />
+        <product-detail-buybox v-if="product !== null" :data-product="product" :data-options="options" />
```

Comment:
Not necessary, see comment below

Reply:
Fixed in [c1daf44](https://github.com/hubblecommerce/hubble-frontend-pwa/commit/c1daf445cf493dd018143e9280033f4d83f82b2f)